src/main/java/com/thatredhead/redbot/command/impl/EvalCommands.java

```diff
@@ -0,0 +1,48 @@
+package com.thatredhead.redbot.command.impl;
+
+import com.thatredhead.redbot.command.Command;
+import com.thatredhead.redbot.command.CommandGroup;
+import com.thatredhead.redbot.helpers4d4j.MessageParser;
+import com.thatredhead.redbot.helpers4d4j.Utilities4D4J;
+import com.thatredhead.redbot.permission.PermissionContext;
+import org.jetbrains.kotlin.script.jsr223.KotlinJsr223JvmDaemonLocalEvalScriptEngineFactory;
+import sx.blah.discord.util.EmbedBuilder;
+
+import javax.script.ScriptException;
+import java.util.ArrayList;
+
+public class EvalCommands extends CommandGroup {
```

Comment:
For single command, there is no need for a command group. Instead put this in the SystemCommands command group, since it makes more sense there.